<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MixerWidget</class>
 <widget class="QWidget" name="MixerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelPresets">
       <property name="text">
        <string>Presets:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboPresets">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxHighlights">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Highlights</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>1</number>
        </property>
        <item row="0" column="5" rowspan="2">
         <widget class="Slider" name="sliderHLuma">
          <property name="toolTip">
           <string>Value</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
border-radius:3px;</string>
          </property>
          <property name="maximum">
           <number>170</number>
          </property>
          <property name="value">
           <number>85</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="2" colspan="2">
         <widget class="YUVFrame" name="frameHighlights">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="1">
           <widget class="SpinBox" name="spinBoxHR">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background:red;
color:white;</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>-512</number>
            </property>
            <property name="maximum">
             <number>511</number>
            </property>
            <property name="value">
             <number>85</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="SpinBox" name="spinBoxHG">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background:green;
color:white;</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>-512</number>
            </property>
            <property name="maximum">
             <number>511</number>
            </property>
            <property name="value">
             <number>85</number>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="SpinBox" name="spinBoxHB">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background:blue;
color:white;</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>-512</number>
            </property>
            <property name="maximum">
             <number>511</number>
            </property>
            <property name="value">
             <number>85</number>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="labelHB">
            <property name="text">
             <string>&lt;div style=&quot; color:#0000ff;&quot;&gt;Blue&lt;/div&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="labelHG">
            <property name="text">
             <string>&lt;div style=&quot; color:green;&quot;&gt;Green&lt;/div&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelHR">
            <property name="text">
             <string>&lt;div style=&quot; color:#ff0000;&quot;&gt;Red&lt;/div&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="1">
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutWeight">
       <property name="spacing">
        <number>12</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonLocked">
         <property name="maximumSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Lock shadows to highlights</string>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/resources.qrc">
           <normaloff>:/actions/Icons/actions/open.png</normaloff>
           <normalon>:/actions/Icons/actions/closed.png</normalon>:/actions/Icons/actions/open.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelWeight">
         <property name="text">
          <string>Weight</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Slider" name="sliderWeight">
         <property name="toolTip">
          <string>Adjust prevalent color</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="SpinBox" name="spinBoxWeight">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxShadows">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Shadows</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>1</number>
        </property>
        <item row="0" column="4" rowspan="2">
         <widget class="Slider" name="sliderSLuma">
          <property name="toolTip">
           <string>Value</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
border-radius:3px;</string>
          </property>
          <property name="maximum">
           <number>170</number>
          </property>
          <property name="value">
           <number>85</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2" colspan="2">
         <widget class="YUVFrame" name="frameShadows">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="1">
           <widget class="SpinBox" name="spinBoxSR">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QSpinBox {
	background:red;
	color:white;
}
QSpinBox:!enabled {
	background:#995555;
}
</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>-512</number>
            </property>
            <property name="maximum">
             <number>511</number>
            </property>
            <property name="value">
             <number>110</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="SpinBox" name="spinBoxSG">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QSpinBox {
	background:green;
	color:white;
}
QSpinBox:!enabled {
	background:#559955;
}
</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>-512</number>
            </property>
            <property name="maximum">
             <number>511</number>
            </property>
            <property name="value">
             <number>148</number>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="SpinBox" name="spinBoxSB">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QSpinBox {
	background:blue;
	color:white;
}
QSpinBox:!enabled {
	background:#555599;
}
</string>
            </property>
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>-512</number>
            </property>
            <property name="maximum">
             <number>511</number>
            </property>
            <property name="value">
             <number>253</number>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="labelSB">
            <property name="styleSheet">
             <string notr="true">QLabel {
	color:blue;
}
QLabel:!enabled {
	color:#404080;
}
</string>
            </property>
            <property name="text">
             <string>Blue</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="labelSG">
            <property name="styleSheet">
             <string notr="true">QLabel {
	color:green;
}
QLabel:!enabled {
	color:#408040;
}
</string>
            </property>
            <property name="text">
             <string>Green</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelSR">
            <property name="styleSheet">
             <string notr="true">QLabel {
	color:red;
}
QLabel:!enabled {
	color:#804040;
}
</string>
            </property>
            <property name="text">
             <string>Red</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="7" column="1">
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>YUVFrame</class>
   <extends>QFrame</extends>
   <header>ui/yuvframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Slider</class>
   <extends>QSlider</extends>
   <header>ui/slider.hpp</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>ui/spinbox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
